community/kernel/src/docs/ops/cache.asciidoc

```diff
@@ -15,34 +15,19 @@ The object cache caches the nodes, relationships and properties in a format that
+* There is just one setting for the file buffer cache: `dbms.pagecache.memory` - it specifies how much memory Neo4j is allowed to use for this cache.
 ***********
 
 The file buffer cache caches the Neo4j data in the same format as it is represented on the durable storage media.
 The purpose of this cache layer is to improve both read and write performance.
-The file buffer cache improves write performance by writing to the cache and deferring durable write until the logical log is rotated.
-This behavior is safe since all transactions are always durably written to the logical log, which can be used to recover the store files in the event of a crash.
+The file buffer cache improves write performance by writing to the cache and deferring durable write.
+This behavior is safe since all transactions are always durably written to the transaction log, which can be used to recover the store files in the event of a crash.
 It also improves write performance by batching up many small writes into fewer page-sized writes.
 
-Since the operation of the cache is tightly related to the data it stores, a short description of the Neo4j durable representation format is necessary background.
-Neo4j stores data in multiple files and relies on the underlying file system to handle this efficiently.
-Each Neo4j storage file contains uniform fixed size records of a particular type:
-
-include::includes/record_sizes_table.asciidoc[]
-
-For strings and arrays, where data can be of variable length, data is stored in one or more 120B chunks, with 8B record overhead.
-The sizes of these blocks can actually be configured when the store is created using the `string_block_size` and `array_block_size` parameters.
-The size of each record type can also be used to calculate the storage requirements of a Neo4j graph or the appropriate cache size for each file buffer cache.
-Note that some strings and arrays can be stored without using the string store or the array store respectively, see <<short-strings>> and <<short-arrays>>.
-
-Neo4j uses multiple file buffer caches, one for each different storage file.
-Each file buffer cache divides its storage file into a number of equally sized windows.
-Each cache window contains an even number of storage records.
-The cache holds the most active cache windows in memory and tracks hit vs. miss ratio for the windows.
-When the hit ratio of an uncached window gets higher than the miss ratio of a cached window, the cached window gets evicted and the previously uncached window is cached instead.
-
-IMPORTANT: Note that the block sizes can only be configured at store creation time.
+Since the file buffer cache is caching the contents of the store files, you can find out how big a page cache you need to keep everything in memory, by summing up the space usage of all the store files.
```

Comment:
Since the file buffer cache is caching the contents of the store files, you can calculate the appropriate size for it by summing up the space usage of all the store files.